Packs/RSANetWitness_v11_1/Layouts/layoutscontainer-RSA_NetWitness_Layout.json

```
@@ -565,6 +525,32 @@
                 "id": "canvas",
                 "name": "Canvas",
                 "type": "canvas"
+            },
+            {
+                "hidden": false,
+                "id": "lwg9usylyq",
```

```suggestion
                "id": "Alerts",
```
please check manually

---

Done